### ISSUE TITLE
delete_cgroup requires binary parameter. is this intended behaviour?

### ISSUE THREAD

**Issue by Skamander** [cloudsigma/cgroupspy#7]
hi,

don't really have experience with python, but since I'm browsing cgroup libraries at the moment I noticed the following: if I create a cgroup I can pass a string as the group name to the `create_cgroup` function, and the name is encoded before it's used, but if I want to delete the same cgroup later with `delete_cgroup` I need to pass it a binary string b"..." otherwise I get a TypeError (can't mix strings and bytes in path components).

Is this intended behaviour or a missing encode before [join](https://github.com/cloudsigma/cgroupspy/blob/master/cgroupspy/nodes.py#L156)?

EDIT: it's the same error as in this commit: https://github.com/cloudsigma/cgroupspy/commit/9baa994f34b1dc41fe22bf8fea7e2c39a5302b6f

Example

``` python
from cgroupspy import trees

t = trees.Tree()
# passing a string works
t.root.create_cgroup("testcgroupspy")

# type error
t.root.delete_cgroup("testcgroupspy")

# works
t.root.delete_cgroup(b"testcgroupspy")
```

**Comment by jjlopez** [comment]
Definitely it's not the expected behavior.
Btw thanks for your contribution.
I'll check and merge your pull request.

**Comment by jjlopez** [comment]
I merged the changes in the [pull request #8](https://github.com/cloudsigma/cgroupspy/pull/8) that fix this issue.
